File: mobile/src/main/java/tr/xip/prayertimes/ui/app/NamazVakitleriApplication.java
package tr.xip.prayertimes.ui.app;

import android.app.Application;
import android.content.Context;

import tr.xip.prayertimes.api.DiyanetApi;
import tr.xip.prayertimes.db.DatabaseManager;

public class NamazVakitleriApplication extends Application {

    private static Context sContext;

    @Override
    public void onCreate() {
        super.onCreate();
        sContext = this;
        DiyanetApi.init();
        DatabaseManager.init();
    }

    public static Context getContext() {
        return sContext;
    }
}

File: mobile/src/main/kotlin/tr/xip/prayertimes/app/NamazVakitleriApplication.kt
package tr.xip.prayertimes.app

import android.app.Application
import android.content.Context

import tr.xip.prayertimes.client.DiyanetClient
import tr.xip.prayertimes.db.DatabaseManager

class NamazVakitleriApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        context = this
        DiyanetClient.init()
        DatabaseManager.init()
    }

    companion object {
        var context: Context? = null
            private set
    }
}